scripts/mirror.js

```diff
@@ -193,6 +191,79 @@ const copyStatement = (data) => {
   return newData;
 };
 
+/**
+ * @param {...SupportStatement} data
+ * @returns {SupportStatement}
+ */
+const combineStatements = (...data) => {
+  const ignored_keys = ['version_added', 'notes'];
```

Comment:
CamelCase for consistency:

```suggestion
  const ignoredKeys = ['version_added', 'notes'];
```